webapp/src/containers/modals/SubscribeModal/index.tsx

```diff
@@ -282,6 +283,48 @@ const SubscribeModal = () => {
             areaPath: newValue,
         });
 
+    const handleSetBuildPipelineFilter = (newValue: string, pipelineName?: string) =>
```

Comment:
`pipelineName` this is not being used anywhere, is this correct?
same for below as well